jekyll/_cci2/workflows.adoc

```diff
@@ -107,6 +107,52 @@ Define job dependencies using the `requires` key. A job must wait until all upst
 
 See the link:https://github.com/CircleCI-Public/circleci-demo-workflows/blob/sequential-branch-filter/.circleci/config.yml[Sample Sequential Workflow config] for a full example.
 
+[#flexible-job-dependency]
+==== Flexible job dependency
+
+Using `requires`, you can build complex dependency maps and fan in/out scenarios.
+
+This example shows a workflow with four sequential jobs. Each job waits until the "required" job attains a specified status.
+
+This configuration snippet is an example of a workflow configured for flexible sequential jobs:
+
+[source,yaml]
+----
+workflows:
+  build-and-deploy:
+    jobs:
+      - build
+      - deploy:
+          requires:
+            - build
+      - release:
+          requires:
+            - deploy
+      - rollback:
+          requires:
+            - release:
+              - failed
+              - canceled
+----
+
+A job must wait until all upstream jobs in the dependency graph have attained either of the statuses specified under the corresponding `requires` key. In this example, the `deploy` job runs when the `build` job completes successfully:
```

Comment:
```suggestion
A job must wait until all upstream jobs in the dependency graph have attained one of the statuses specified under the corresponding `requires` key. In this example, the following dependencies are configured:
```

Reply:
Just a suggestion to remove the duplication of the deploy job run explanation as it's in the bullet list too